plugins/packages/packages_inspector.rb

```diff
@@ -90,12 +90,22 @@ def inspect_dpkg
       )
     end
     packages.each_slice(100) do |packages_slice|
-      apt_cache_output = @system.run_command(
-        "apt-cache",
-        "show",
-        *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
-        stdout: :capture
-      )
+      begin
+        apt_cache_output = @system.run_command(
+          "apt-cache",
+          "show",
+          *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
+          stdout: :capture
+        )
+      rescue
+        # Older Debian systems do not support the version parameter for `apt-cache show`
+        apt_cache_output = @system.run_command(
+          "apt-cache",
+          "show",
+          *packages_slice.map(&:name),
+          stdout: :capture
```

Comment:
Is there a way to not have an exception for every hundred packages? 

This is bad for performance and I find it hard to read, exceptions shouldn't be part of the normal execution flow.
If the performance is no issue, it would be sufficient to extract the `apt_cache_output` assignment to methods to improve readability.

Also this method is too long, it would benefit from some "extract to method". 

Not really part of this PR, but the method names 'inspect_*' do hint at some hidden classes. In the future we might want to split this into a `DpkgInspector` and a `RPMInspector`.